src/test/assets/meson/meson.build

```diff
@@ -0,0 +1,15 @@
+project('meson',
+          version: '0.1.0',
+    meson_version: '>= 0.50.0',
+  default_options: [ 'warning_level=2',
+                   ],
+)
+
+i18n = import('i18n')
+
+
+
+subdir('data')
```

Comment:
I would avoid using subdirs, just put all the necessary files for the test on the same directory.